### PR TITLE
css calc negative

### PR DESCRIPTION
**Pull request by tedk13**

### CODE REVIEW

File: src/content/posts/2023-07-21-negative-calc-values.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- css
+twitter_text: "Here's how you can set a custom property to a negative value using the calc() css function."
+authors: 
+- Ted Krueger
+image: https://plus.unsplash.com/premium_photo-1661382019197-94d5edb38186?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1772&q=80
+image_url: https://unsplash.com/photos/FmzQpb6db2s
+image_credit: Unsplash+
+---
+
+I want to show a quick demo of how you can use custom properties and set them to a negative value. I don't want to bore you with any explaination. This ain't no Pintrest recipe about a great german potato salad. Let's check out the code.
```
Comment (jaime-lynn) [inline src/content/posts/2023-07-21-negative-calc-values.md:20]:
```suggestion
I want to show a quick demo of how you can use custom properties and set them to a negative value. I don't want to bore you with any explanation. This ain't no Pinterest recipe about a great German potato salad. Let's check out the code.
```

File: src/content/posts/2023-07-21-negative-calc-values.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+header {
+  height: var(--header-height);
+}
+
+.other-class-that-needs-negative-something {
+  margin-top: calc(var--header-height * (-1));
+}
+```
+
+_Now_ I'll explain. If you were to try `-#{var(--header-height)}` you would get something like, `margin-top: -var(--header-height)`, which the browser will not know what to do with. You can instead use the `calc()` css function to take your variable and multiply it by `-1` and boom.
```
Comment (jaime-lynn) [inline src/content/posts/2023-07-21-negative-calc-values.md:34]: I'd suggest either
```to take your variable, multiple it by `-1`, and boom.```
or
```to take your variable and multiple it by `-1`. Boom.```

Comment (tedk13) [reply src/content/posts/2023-07-21-negative-calc-values.md:34]: What do you have against `and`?

Comment (jaime-lynn) [reply src/content/posts/2023-07-21-negative-calc-values.md:34]: Isn't it like a thing not to link too many things with `and` so it isn't too much of a run-on? And I'm like the queen of run-on sentences.

**Review by jaime-lynn — APPROVED** [review]

**Pull request merged**